spec/fixtures/proofing/aamva/requests/verification_request.xml

```diff
@@ -31,7 +31,7 @@
           <nc:LocationStateUsPostalServiceCode>VA</nc:LocationStateUsPostalServiceCode>
           <nc:LocationPostalCode>20176</nc:LocationPostalCode>
         </aa:Address>
-      </dldv:verifyDriverLicenseDataRequest>
+      <aa:DocumentCategoryCode>1</aa:DocumentCategoryCode></dldv:verifyDriverLicenseDataRequest>
     </dldv:VerifyDriverLicenseData>
   </soap:Body>
-</soap:Envelope>
\ No newline at end of file
+</soap:Envelope>
```

Comment:
I think this is some DOS vs. UNIX line endings cursedness. There is not an _extra_ newline at the end; it's just a plain text file.